systemtests/tests/python-fd-plugin-postgres-test/python-modules/BareosFdPluginLocalFilesetWithRestoreObjects.py

```diff
@@ -0,0 +1,353 @@
+#!/usr/bin/env python
```

Comment:
Is this file needed anymore? This may be a copy-relict of a sample.

Reply:
You are probably right but this file is not part of the PR?

Reply:
This file is part of this PR, because it appears in the file list.